cmd/ike/main.go

```diff
@@ -10,7 +10,7 @@ func main() {
 	// implementing the logr.Logger interface. This logger will
 	// be propagated through the whole operator, generating
 	// uniform and structured logs.
-	logf.SetLogger(logf.ZapLogger(false))
+	logf.SetLogger(logf.ZapLogger(true))
```

Comment:
Need a better way to handle this as we want the Service to deploy using 'false'

Reply:
You want that to be fixed as part of this PR? Maybe we need two separated things - CLI and that thing which runs on the server? That's probably a decision which needs to be discussed in a bit more detail. For now I'm fine with `json` output - can revert it to the previous state and make an issue for the potential split. WDYT?

Reply:
sounds good.